src/Tools/AI Test Toolkit/src/Logs/AITLogEntries.Page.al

```diff
@@ -253,6 +270,19 @@ page 149033 "AIT Log Entries"
                     Page.Run(Page::"AIT Test Data Compare", Rec);
                 end;
             }
+            action("Export Results")
+            {
+                Caption = 'Export Results';
+                Image = Export;
+                ToolTip = 'Exports the results.';
```

Comment:
Can you provide a better tool tip. It might be confusing with the existing action of download test output.